Makefile.include

```diff
@@ -110,6 +113,20 @@ gen-alertmanager:     # Generate Alertmanager client.
 clean_swagger:
 	find api -name '*.swagger.json' -print -delete
 
+
+ifeq ($(shell test "${SWAGGER_UI_VERSION}" = "latest" && echo 1 || echo 0), 1)
+get_swagger_version:
+override SWAGGER_UI_VERSION = $(shell curl --silent --head https://github.com/swagger-api/swagger-ui/releases/latest | grep -E '^[lL]ocation' | sed 's;^.*/;;')
+else
+get_swagger_version:
+endif
+
+update-swagger: get_swagger_version               ## Update Swagger UI in api/swagger, use `SWAGGER_UI_VERSION=xxx make update-swagger` to choose a version other than latest.
```

Comment:
Renamed this to match the format of the other recipes that are show via help, where a hypen is used instead of underscores